agent/taskresource/cgroup/cgroup.go

```diff
@@ -244,18 +251,18 @@ func (cgroup *CgroupResource) GetCreatedAt() time.Time {
 func (cgroup *CgroupResource) Create() error {
 	err := cgroup.setupTaskCgroup()
 	if err != nil {
-		seelog.Criticalf("Cgroup resource [%s]: unable to setup cgroup root: %v", cgroup.taskARN, err)
+		cgroup.log.Errorf("unable to setup cgroup root: %v", err)
 		return err
 	}
 	return nil
 }
 
 func (cgroup *CgroupResource) setupTaskCgroup() error {
 	cgroupRoot := cgroup.cgroupRoot
-	seelog.Debugf("Cgroup resource [%s]: setting up cgroup at: %s", cgroup.taskARN, cgroupRoot)
+	cgroup.log.Info("setting up cgroup")
```

Comment:
we should keep identification information in the logs, to help debugging (here and other places).

Reply:
got updated on the design, dismiss this comment please.